zerver/tests/test_push_notifications.py

```diff
@@ -443,12 +443,13 @@ def test_register_remote_push_user_params(self) -> None:
         )
         self.assert_json_error(result, "Must validate with valid Zulip server API key")
 
-        result = self.uuid_post(
-            self.server_uuid,
-            endpoint,
-            dict(user_id=user_id, token_kind=token_kind, token=token),
-            subdomain="zulip",
-        )
+        with self.settings(DEVELOPMENT=False):
```

Comment:
Should this still be overriding `DEVELOPMENT` rather than `DEVELOPMENT_DISABLE_PUSH_BOUNCER_DOMAIN_CHECK`? And do we want to set that in `test_settings.py`?

Reply:
Oops, thanks for catching - I think this commit shouldn't be modifying the test at all anymore. I wouldn't set `DEVELOPMENT_DISABLE_PUSH_BOUNCER_DOMAIN_CHECK = True` in `test_settings`, because our tests are written with the domain check being active in mind - e.g. this original test that was modifying here, which tests the check.

My current idea here was to just have this `DEVELOPMENT_DISABLE_PUSH_BOUNCER_DOMAIN_CHECK` line in `dev_settings` that we can quickly uncomment when we want to do simple bouncer testing in dev env - and to not change any current behaviors by default.

Reply:
OK, that works.